lints/0024_permissive_rls_policy.sql

```diff
@@ -0,0 +1,124 @@
+create view lint."0024_permissive_rls_policy" as
+
```

Comment:
lets rename this to "rls_policy_always_true" since `permissive` has a specific meaning in RLS